src/main/java/org/opentripplanner/model/plan/Leg.java

```diff
@@ -184,7 +183,7 @@
 
     public Boolean rentedVehicle;
 
-   public List<String> vehicleRentalNetworks = new ArrayList<>();
+    public String vehicleRentalNetwork;
```

Comment:
I would be the right thing to do but does this change the REST API?

Reply:
No, that class is an internal class, [this](https://github.com/entur/OpenTripPlanner/blob/otp2_feed_scoped_vehicle_rental_station/src/main/java/org/opentripplanner/api/model/ApiLeg.java) is the model for the REST API